<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentDialog</class>
 <widget class="QDialog" name="PaymentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Process Payment</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog { background-color: #F8FAFC; font-family: 'Segoe UI', sans-serif; }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_header">
     <property name="styleSheet">
      <string>background-color: white; border-radius: 12px; border: 1px solid #E2E8F0;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_header">
      <item>
       <widget class="QLabel" name="label_title">
        <property name="styleSheet">
         <string>color: #64748B; font-size: 12px; font-weight: 700; letter-spacing: 1px; border: none;</string>
        </property>
        <property name="text">
         <string>TOTAL AMOUNT DUE</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_total_amount">
        <property name="styleSheet">
         <string>font-size: 48px; font-weight: 800; color: #0F172A; border: none;</string>
        </property>
        <property name="text">
         <string>₱0.00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_vat_breakdown">
        <property name="styleSheet">
         <string>color: #94A3B8; font-size: 11px; border: none;</string>
        </property>
        <property name="text">
         <string>VAT Included: ₱0.00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_method">
     <property name="styleSheet">
      <string>color: #64748B; font-size: 11px; font-weight: 700; margin-top: 10px;</string>
     </property>
     <property name="text">
      <string>SELECT PAYMENT METHOD</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_methods">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_cash">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>51</height>
        </size>
       </property>
       <property name="styleSheet">
        <string>
         QPushButton { background-color: white; border: 1px solid #CBD5E1; border-radius: 8px; font-weight: 600; color: #334155; }
         QPushButton:checked { background-color: #E0F2FE; border: 2px solid #0891B2; color: #0891B2; }
        </string>
       </property>
       <property name="text">
        <string>Cash</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_gcash">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>51</height>
        </size>
       </property>
       <property name="styleSheet">
        <string>
         QPushButton { background-color: white; border: 1px solid #CBD5E1; border-radius: 8px; font-weight: 600; color: #334155; }
         QPushButton:checked { background-color: #E0F2FE; border: 2px solid #0891B2; color: #0891B2; }
        </string>
       </property>
       <property name="text">
        <string>GCash</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_card">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>51</height>
        </size>
       </property>
       <property name="styleSheet">
        <string>
         QPushButton { background-color: white; border: 1px solid #CBD5E1; border-radius: 8px; font-weight: 600; color: #334155; }
         QPushButton:checked { background-color: #E0F2FE; border: 2px solid #0891B2; color: #0891B2; }
        </string>
       </property>
       <property name="text">
        <string>Card</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="input_tendered">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>56</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
       QLineEdit { color: rgb(74, 74, 74); background-color: white; border: 1px solid #CBD5E1; border-radius: 8px; padding: 0 15px; font-size: 18px; font-weight: bold; }
       QLineEdit:focus { border: 2px solid #0891B2; }
      </string>
     </property>
     <property name="placeholderText">
      <string>Enter Amount </string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_quick_cash">
     <item>
      <widget class="QPushButton" name="btn_exact">
       <property name="styleSheet">
        <string>QPushButton { background: #F1F5F9; border: none; border-radius: 5px; padding: 8px; font-weight: bold; color: #475569; } QPushButton:hover { background: #E2E8F0; }</string>
       </property>
       <property name="text">
        <string>Exact</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_100">
       <property name="styleSheet">
        <string>QPushButton { background: #F1F5F9; border: none; border-radius: 5px; padding: 8px; font-weight: bold; color: #475569; } QPushButton:hover { background: #E2E8F0; }</string>
       </property>
       <property name="text">
        <string>+ ₱100</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_500">
       <property name="styleSheet">
        <string>QPushButton { background: #F1F5F9; border: none; border-radius: 5px; padding: 8px; font-weight: bold; color: #475569; } QPushButton:hover { background: #E2E8F0; }</string>
       </property>
       <property name="text">
        <string>+ ₱500</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_1000">
       <property name="styleSheet">
        <string>QPushButton { background: #F1F5F9; border: none; border-radius: 5px; padding: 8px; font-weight: bold; color: #475569; } QPushButton:hover { background: #E2E8F0; }</string>
       </property>
       <property name="text">
        <string>+ ₱1000</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="input_reference">
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string>color: rgb(74, 74, 74); background-color: white; border: 1px solid #CBD5E1; border-radius: 12px; padding: 0 15px; font-size: 14px;</string>
     </property>
     <property name="placeholderText">
      <string>Enter Ref # (Proof of Payment)</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="vspacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="frame_change">
     <property name="styleSheet">
      <string>background-color: #F0FDF4; border-radius: 10px; border: 1px dashed #86EFAC;</string>
     </property>
     <layout class="QHBoxLayout" name="hbox_change">
      <item>
       <widget class="QLabel" name="label_change">
        <property name="styleSheet">
         <string>font-size: 14px; font-weight: 700; color: #15803D; border: none;</string>
        </property>
        <property name="text">
         <string>CHANGE DUE:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_change_amount">
        <property name="styleSheet">
         <string>font-size: 24px; font-weight: 800; color: #15803D; border: none;</string>
        </property>
        <property name="text">
         <string>₱0.00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="vspacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_actions">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string>QPushButton { background-color: white; color: #64748B; border: 1px solid #CBD5E1; border-radius: 10px; font-weight: 700; } QPushButton:hover { color: #EF4444; border-color: #EF4444; background-color: #FEF2F2; }</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_confirm">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string>QPushButton { background-color: #0891B2; color: white; border-radius: 10px; font-weight: 800; border: none; } QPushButton:disabled { background-color: #E2E8F0; color: #94A3B8; }</string>
       </property>
       <property name="text">
        <string>CONFIRM PAYMENT</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
